vdplus/api/google/__init__.py

```diff
@@ -6,17 +6,29 @@
 
 def _google_creds_fn():
     from pkg_resources import resource_filename
-    return resource_filename('vdplus.api.google', 'google-creds.json')
+    import os
+    if not os.path.exists(Path(resource_filename('vdplus.api.google', 'google-creds.json'))):
+        vd.error('google-creds.json file does not exist. create it by following this guide: https://github.com/saulpw/visidata/blob/develop/docs/gmail.md')
```

Comment:
I'm not sure how I can reference the URL in a more dynamic (context specific) way?

Reply:
This should show up at https://visidata.org/docs/google eventually; is that what you mean?

Reply:
Ah, yes. That is probably the best place to reference the docs at. Thanks.